apps/src/blockTooltips/DropletFunctionTooltip.html.ejs

```diff
@@ -22,11 +22,17 @@
     </div>
   <% }) %>
 <% } %>
-<% var linkText = "See examples" %>
-<% if (locals.customExamplesLink) { %>
-  <% linkText = "" %>
+<% var linkText = "" %>
+<% if (locals.showCodeLink) { %>
+  <% linkText = "Show code" %>
+  <div class="tooltip-code-link">
+    <a href="javascript:void(0);">
+      <%= linkText %>
```

Comment:
nit: this can be simplified by using the string literal directly. I think `<% var linkText = "" %>`